test/fixtures/basic/public/测试.md

```
@@ -0,0 +1,3 @@
+# Chinese Heading
+
+这是一个中文测试文件。
```

markdownlint-cli2 (0.23.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)